tests/system/src/test/java/test/robot/javafx/scene/tableview/TableViewResizeColumnToFitContentTest.java

```diff
@@ -97,11 +94,14 @@ public void resizeColumnToFitContentTest() {
         }
         Assert.assertTrue("resizeColumnToFitContent failed",
                 (colTwoWidth != table.getColumns().get(1).getWidth()));
-        colTwoWidth = table.getColumns().get(1).getWidth();
-        colThreeWidth = table.getColumns().get(2).getWidth();
-        double colsWidthAfterResize = colOneWidth + colTwoWidth + colThreeWidth;
-        Assert.assertEquals("TableView.CONSTRAINED_RESIZE_POLICY ignored.",
-                colsWidthBeforeResize, colsWidthAfterResize, 0);
+
+        if (!Util.isFractionalScale(table)) {
```

Comment:
Please add a comment with the bug ID of the bug that is the reason for skipping this part of the test. Something like:

```
    // Skip this check on platforms with fractional scale until JDK-8305709 is fixed
```